test/NavbarSpec.js

```diff
@@ -61,7 +61,7 @@ describe('Nav', function () {
     let instance = ReactTestUtils.renderIntoDocument(
       <Navbar componentClass={'header'}/>
     );
-    assert.ok(React.findDOMNode(instance).nodeName, 'HEADER');
+    assert.equal(React.findDOMNode(instance).nodeName, 'HEADER');
```

Comment:
The separate commit is perfect, no need for a seperate PR, thanks for taking the time